src/data/customer/__test__/Config.js

```diff
@@ -30,6 +30,10 @@ const CONFIG = {
   customerId: 14445,
   fuzzy: false,
   fuzziness: 1,
+  jobTest: {
+    orderIds: [45357, 45358],
```

Comment:
@vinay3206 what happened to our discussion where you would create orders in the test first?